streaming_jupyter_integrations/magics.py

```diff
@@ -136,7 +136,7 @@ def flink_execute_sql(self, line: str, cell: str) -> None:
     @__interrupt_signal_decorator
     def __flink_execute_sql_internal(self, stmt: str) -> None:
         task = self.__internal_execute_sql(stmt)
-        if is_dml(stmt) or is_query(stmt):
```

Comment:
why did you remove is_query here?

Reply:
In the previous version of the code `SELECT` was not considered part of DML. In this change all DQLs (so not only `SELECT`, but `DESCRIBE` and `EXPLAIN` too) are considered a subset of DMLs, so there's no point in checking it twice.
`is_query` is still a think in other magic though, as `INSERT` is a DML-that's-not-query and it is not returning anything to show.

Reply:
In the previous version of the code `SELECT` was not considered part of DML. In this change all DQLs (so not only `SELECT`, but `DESCRIBE` and `EXPLAIN` too) are considered a subset of DMLs, so there's no point in checking it twice.
`is_query` is still a thing in other magic though, as `INSERT` is a DML-that's-not-query and it is not returning anything to show.

Of course, we can have a conversation whether DQLs should be considered a subset of DML at all. IMO it can be — it does not hurt us right now and is semantically correct.

Reply:
There is a difference between `is_dml` and `is_dql` now :)